### PR TITLE
Remove interfaces and simplify activity usage

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: node/node-sdk.md [hunk 1]
```diff
@@ -135,39 +109,38 @@ Workflow code must export a `main` function which will be used as the handler me
...
 ### Example
 ```typescript
+// workflows/src/greeter.ts
+
 import ms from 'ms';
 import { Workflow } from '@temporal-sdk/workflow';
-import { GreeterService } from '@interfaces/activities';
+import { greet } from '@activites/greeter';
+
+// Local activity
```
Comment (mfateev) [inline node/node-sdk.md:125]: Local activities should follow exactly the same pattern for definition and invocation as normal ones.

File: node/node-sdk.md [hunk 2]
```diff
@@ -135,39 +109,38 @@ Workflow code must export a `main` function which will be used as the handler me
...
+const greetWithCustomTimeout = Workflow.activity(greet, {
+  startToCloseTimeoutMillis: ms('1 hour'),
+});
 
-const greeter = Workflow.newActivityStub<GreeterService>('GreeterService', {
+// Remote activity
+type httpGet = (url: string) => Promise<any>;
+const greetWithCustomTimeout = Workflow.activity<httpGet>('httpGet', {
```
Comment (mfateev) [inline node/node-sdk.md:132]: I would rename "activity" to newActivityStub and newLocalActivityStub.

**Review by sw-yx — APPROVED** [review]
looking good!

File: node/node-sdk.md (hunk 2 above)
Comment (rylandg) [reply node/node-sdk.md:132]: stub is not a common term in JavaScript world

**Comment by bergundy** [comment]
I merging this into the main proposal, we'll continue the discussion there.

**Pull request merged**